units/src/locktime/relative.rs

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: CC0-1.0
 
-//! Provides type `Height` and `Time` types used by the `rust-bitcoin` `relative::LockTime` type.
+//! Provides [`Height`] and [`Time`] types used by the `rust-bitcoin` [`relative::LockTime`](crate::locktime) type.
```

Comment:
Why is `relative::LockTime` not in this crate?

Reply:
Because I argued that its not a unit but a primitive, IIRC @apoelstra was not too fussed either way. Less philosophically see the list of imports in `bitcoin::locktime::absolute` and `bitcoin::ocktime::relative`, they cannot be trivially moved to `units`.

Reply:
IMO it's more like `Sequence` is the primitive. But Let's keep it as-is, we can move it any time we want.

Reply:
Removed `relative::LockTime` links from relative.rs and absolute.rs.

Reply:
Agreed philosophically that `Sequence` is a primitive, `relative::LockTime` is a unit ... and as for `absolute::LockTime` who can say?

But as I recall the decision was actually made pragmatically because of difficulties moving the types.

Anyway this is not the place to litigate this :).

Reply:
I'll comment for Jamil "STOP DERAILING MY THREAD" :)